.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
Ok, that makes sense. Yeah, if it's only for CI, I'm fine with leaving it here. No need to burden devs with more time wasted downloading packages :smile: